tools/cldr-apps/src/main/java/org/unicode/cldr/web/DataPage.java

```diff
@@ -2129,7 +2083,7 @@ private void populateFromThisXpath(String xpath, Set<String> extraXpaths, CLDRFi
             checkCldr.getExamples(xpath, isExtraPath ? null : ourValue, examplesResult);
         }
         if (ourValue != null && ourValue.length() > 0) {
-            addOurValue(ourValue, row, checkCldrResult, sourceLocaleStatus, xpath, setInheritFrom);
+            addOurValue(ourValue, row, checkCldrResult, sourceLocaleStatus, xpath);
```

Comment:
IntelliJ pointed out that setInheritFrom was always null here and so had no effect in addOurValue. It's true though not very obvious. setInheritFrom is null if ourValueIsInherited is false. So for setInheritFrom to be non-null, ourValueIsInherited must be true. But then we would have returned already if (ourValueIsInherited && !isExtraPath), so isExtraPath would need to be true for setInheritFrom to be non-null here. But if isExtraPath is true, then ourValue is null, so addOurValue wouldn't be called, QED